examples/resources/spacelift_stack/resource.tf

```diff
@@ -164,3 +164,21 @@ resource "spacelift_stack" "ansible-stack" {
   repository   = "ansible-playbooks"
   runner_image = "public.ecr.aws/spacelift/runner-ansible:latest"
 }
+
+# Terragrunt stack using github.com as VCS
+resource "spacelift_stack" "terragrunt-stack" {
+  terragrunt {
+    terraform_version      = "1.6.2"
```

Comment:
are we sure we want to call it terraform version?

Reply:
It's not a new field, so I can't really change it without breaking backwards compatibility. I could introduce a new field and deprecate it, but I'd rather not add that complication.

Reply:
yeah but wouldn't it make more sense to add an open tofu version?

- if both populated thrown an error
- if opentofu version populated, set the tool to tofu
- tf version is populated, set the tool to tf

Reply:
I don't think that would work for a couple of reasons:

1. We support defaulting to the latest version if you don't specify it. So we'd still need a separate `tool` field to indicate what you wanted to use in that situation.
2. We support manual provisioning, where no OpenTofu / Terraform version is specified.

Reply:
We could add an `opentofu_version` field or `tool_version`, and just allow either to be used. Not sure I love having two fields for the same thing though :disappointed:

Reply:
keep it as is